ga4gh/datamodel/ontologies.py

```diff
@@ -0,0 +1,82 @@
+"""
+Support for Ontologies.
+FIXME: this is currently hard-coded for the GENCODE set as a short-term hack
+to support BRCA changeling.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import ga4gh.protocol as protocol
+
+
+class OntologyTermSet(object):
+    """A set of related ontology terms.
+    """
+    def __init__(self, ontologySource):
+        self.ontologySource = ontologySource
+        self.byId = dict()
```

Comment:
It would be nice to follow the conventions used elsewhere for these maps. Something like ``self._ontologyTermIdMap`` (or just ``self._idMap`` if this is too verbose) would be good.